package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "app",
+  "private": "true",
+  "dependencies": {
+    "@hotwired/stimulus": "^3.2.2",
+    "@hotwired/turbo-rails": "^7.3.0",
+    "webpack": "^5.88.2",
+    "webpack-cli": "^5.1.4"
+  },
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  },
+  "devDependencies": {
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
+}
```

Comment:
- [ ] Your project is missing some dependencies such as react-dom and even redux because I can see you are using a local state. I think you did not use CRA which is why the react-dom is not in these dependencies, in fact, the script does not have the start command, I will encourage you to make your dependencies look like the following:
```
{
  "name": "app",
  "private": "true",
  "scripts": {
    "start": "rails server",
    "build": "webpack --config webpack.config.js"
  },
  "dependencies": {
    "@babel/core": "^7.22.10",
    "@babel/preset-env": "^7.22.10",
    "@babel/preset-react": "^7.22.5",
    "@reduxjs/toolkit": "^1.9.5",
    "axios": "^1.4.0",
    "babel-loader": "^9.1.3",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-redux": "^8.1.2",
    "react-router-dom": "^6.15.0",
    "react-scripts": "^5.0.1"
  },
  "devDependencies": {
    "stylelint": "^13.13.1",
    "stylelint-config-standard": "^21.0.0",
    "stylelint-csstree-validator": "^1.9.0",
    "stylelint-scss": "^3.21.0"
  }
}
```